development/language/guidelines.rst

```diff
@@ -0,0 +1,190 @@
+                          icon_post_quote.gif
+                          icon_post_report.gif
+                          icon_user_offline.gif
+                          icon_user_online.gif
+                          icon_user_profile.gif
+                          icon_user_search.gif
+                          icon_user_warn.gif
+                          index.htm (optional)
+                          stylesheet.css
+
+5) Submissions have to follow as much as possible the `3.1 Translation (i18n/L10n) Guidelines`_ recommendations, especially the `3.1 Writing style`_.
+
+6) All the PHP and text files have to be encoding in UTF-8 without BOM and a new line at the end of the file. Many modern text editors use this as a default setting, but we recommend checking it in your editor's settings. We recommend you `Notepad++`_ or `PSPad`_, both lightweight and free.
+
+7) The translation is mostly your work and you have a right to hold a copyright and names to it.
+
+8) A maximum of 3 links can be included as an author credit in the footer, customisable via the ``'TRANSLATION_INFO'`` key in ``common.php``. Please note that the Translations Manager has complete discretion on what is acceptable as an author credit link.
+
+9) Submissions have to be submitted as a single zip file. For information, the Customisation Database will automatically name ``languagename_versionnumber.zip`` your uploaded language pack. For example, if a Brazilian Portuguese language pack author uploads an archive named ``Brasileiro_1.0.5.zip``, it will be automatically changed to ``brazilian_portuguese_1_0_5.zip``.
+
+10) Contribution description in the Customisation Database should be translated into English in addition of your local language. Some administrators might want to download your translation without speak your language.
+
+11) Contribution screenshot in the Customisation Database should only be the flag of the country whose the language is spoken. For example, the flag of France for the French language.
+
+12) Revision name in the Customisation Database should be left blank, contain the phpBB package version and/or package release name (e.g. "**3.0.12 / Richard 'D¡cky' Foote**" for 3.0.12) for more understanding.
+
+13) The Demo URL in the Customisation Database must be empty, unless you want to put a link to an international community (`officially`_ listed or not) related to the language of the contribution. For example, http://www.phpbbarabia.com/ as Demo URL concerning the `Arabic language`_ is allowed.
+
+.. _Customisation Database: https://www.phpbb.com/go/customise/language-packs/3.1
+.. _Language Packs Database: https://www.phpbb.com/languages/
+.. _GNU General Public License 2.0: http://www.opensource.org/licenses/gpl-2.0.php
+.. _3.1 Translation (i18n/L10n) Guidelines: https://area51.phpbb.com/docs/31x/coding-guidelines.html#translation
+.. _3.1 Writing style: https://area51.phpbb.com/docs/31x/coding-guidelines.html#writingstyle
```

Comment:
Yes, but only in the 3.2.x branch,  this PR is against 3.1.x, 3.2.x update will come, when this PR is merged upstream.